db/migrate/20170309152057_add_data_to_users.rb

```diff
@@ -0,0 +1,6 @@
+class AddDataToUsers < ActiveRecord::Migration[5.0]
+  def change
+    add_column :users, :gender, :string, default: 'Not defined'
```

Comment:
Hmm, a few moments before I saw a code which assings "Not defined" value in a before filter if the value is nil. And here we have a constraint defined which gives the same result. Why there is a need for such redundancy?